pkg/plugin/conftest/plugin_test.go

```diff
@@ -402,7 +402,7 @@ func TestPlugin_GetScanJobSpec(t *testing.T) {
 		"Affinity":                     Equal(starboard.LinuxNodeAffinity()),
 		"Volumes": ConsistOf(
 			MatchFields(IgnoreExtras, Fields{
-				"Name": Equal("scan-configauditreport-5d4445db4f-volume"),
+				"Name": Equal("scan-configauditreport-789cbb5cc4-volume"),
```

Comment:
No reason to update the test name that I can see.

Reply:
Also here, the Name property is the name of a scan job not a test case.